### DISCUSSION TITLE
liboqs different version of openssl

### DISCUSSION THREAD

**Discussion by dominik19990** [open-quantum-safe/liboqs#1752] [Q&A]
Hey, I'd like to use the liboqs. My Ubuntu has pre-installed openssl in verison 3.0.2, but I've installed 3.2.1 version. But every time i run the cmake -GNinja command, it takes the older 3.0.2 version. How can I use the newer one? I tried to somehow edit the CMakeLists.txt, but it still takes the older version. Thanks for answer

**Comment by baentsch** [comment]
Please read the documentation of [OQS_USE_OPENSSL](https://github.com/open-quantum-safe/liboqs/blob/main/CONFIGURE.md#oqs_use_openssl), particularly on OPENSSL_ROOT_DIR
